pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -713,6 +722,43 @@ type Matcher struct {
 	Regex bool `json:"regex,omitempty"`
 }
 
+// MuteTimeIntercal specifies a named interval of time that may be referenced
+// in the routing tree to mute particular routes for particular times of the day
+type MuteTimeInterval struct {
+	// Name of a Mute Time Interval
+	// +kubebuilder:validation:MinLength=1
+	Name string `json:"name"`
+	// Series of time intervals assoicated with the MuteTimeInterval
+	TimeInterval []TimeInterval `json:"timeInterval"`
```

Comment:
By convention, this variable should be named `TimeIntervals`. The json key should also be pluralized.

Reply:
Ahhhh awesome didn't know this, will update